libs/pages/layout/src/lib/ui/navigation/navigation.tsx

```diff
@@ -22,7 +23,10 @@ export interface NavigationProps {
 
 export function Navigation(props: NavigationProps) {
   const { firstName, lastName, darkMode } = props
-  const { organizationId } = useParams()
+  const { organizationId = '', clusterId = '' } = useParams()
+  const { pathname } = useLocation()
+
+  const matchLogInfraRoute = matchPath(pathname, INFRA_LOGS_URL(organizationId, clusterId))
```

Comment:
Can we factorize both matchLogInfraRoute in a util so that we use always the same one and so that we can test the result of the function?